SendToLogit/README.md

```diff
@@ -0,0 +1,35 @@
+# Send a Log entry to Logit.io
+
+### Purpose
+There is a requirement to push certain events into logit.io (ELK) so that it can be presented on a dashboard in Grafana. 
+This routine has been made generic, so that it can be reused if necessary and additional data points can be simply added.
+
```

Comment:
Also important: the log is structured as json so it will be indexed efficiently in Elasticsearch and allows queries and visualisations in Kibana